pwiz_tools/Skyline/TestFunctional/FullScanGraphTest.cs

```diff
@@ -120,6 +121,23 @@ protected override void DoTest()
             ClickFullScan(517, 1000);
             TestScale(516, 520, 0, 80);
 
+            //Check the rank and annotate functionality if we run in 
+            //onscreen mode
+            SetShowAnnotations(true);
+            if (!Skyline.Program.SkylineOffscreen)
+            {
+                TestAnnotations(new []{ "y4 (rank 9)" });

```

Comment:
This fails in Chinese and Japanese.
I think you should use "Resources.AbstractSpectrumGraphItem_GetLabel_rank__0__" here.
(I think that's the string that the graph code uses too; I might be wrong).